pages/api/events/class/index.ts

```diff
@@ -0,0 +1,91 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from "next";
+import { createClassEvent } from "../../../../lib/database/events";
+import { createCalenderEvent } from "../../../../lib/database/calender";
+import { createCommitment } from "../../../../lib/database/commitments";
+import { CreateClassEvent, ClassEvent, CreateClassEventSchema } from "../../../../models/events";
+import { decode } from "io-ts-promise";
+import { StatusCodes } from "http-status-codes";
+import { rrulestr } from "rrule";
+import { DateTime } from "luxon";
+
+// handles requests to /api/events/class
+const eventHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  switch (req.method) {
+    case "POST": {
+      let newEvent: CreateClassEvent;
+      try {
+        newEvent = await decode(CreateClassEventSchema, req.body);
+      } catch (e) {
+        return res.status(StatusCodes.BAD_REQUEST).json("Fields are not correctly entered");
+      }
+      try {
+        const result = await createClassEvent(
```

Comment:
Since `createClassEvent` returns an array corresponding to 2 values, can we split them up here? So the ID's in one variable and the ID of the class in another. That way, we don't have to do a splice at the bottom.